### PR TITLE
Add support for Navitia fr-se region (South-East of France)

### PR DESCRIPTION
**Pull request by thony8**
based of @kalon33 code (South-West France)
I am still working on tests (6/20 tests failed) : I have the same problem in #64 but others failed tests don't detect an invalid id for "from" or "to" field

### CODE REVIEW

**Comment by schildbach** [comment]
Thanks for your contribution. Can you

- Rename French to France everywhere? I know we have it wrong in the provider you copied from – we will need to fix it there separately.
- Add the license header to your tests.
- Perhaps "Copyright 2016"?
- Rebase on current master.

**Comment by schildbach** [comment]
Please don't merge – always use rebase! Can you rebase on current master?

**Comment by thony8** [comment]
Sorry, I fix it

**Comment by schildbach** [comment]
Thanks looks good. If you think you're ready with the tests let me know and I'll squash & merge.

File: enabler/src/de/schildbach/pte/FranceSouthEastProvider.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			}
+			case FERRY:
+			{
+				// Batcub
+				return new Style(Shape.ROUNDED, Style.parseColor(color), computeForegroundColor(color));
+			}
+			case SUBWAY:
+			{
+				// Toulouse subway (from Tisseo network)
+				return new Style(Shape.ROUNDED, Style.parseColor(color), computeForegroundColor(color));
```
Comment (schildbach) [inline enabler/src/de/schildbach/pte/FranceSouthEastProvider.java]: This code has been copied from the other france provider ... but is it right for this one too?

**Comment by schildbach** [comment]
Merged.

**Pull request closed without merging**